pkg/shared/utilityCollections/search.go

```diff
@@ -0,0 +1,10 @@
+package utilityCollections
+
+func Contains[T comparable](s []T, elem T) bool {
+	for _, a := range s {
+		if a == elem {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
Is this used anywhere? If not, we should delete it

We also often use [samber/lo](https://github.com/samber/lo) for functions like [this](https://github.com/samber/lo#contains)

Reply:
Good catch. I didn't like the process of looping through a collection but this attempt did not work. Will give samber a try next time.